zerver/data_import/slack_message_conversion.py

```diff
@@ -79,52 +79,42 @@ def get_user_full_name(user: ZerverFieldsT) -> str:
         return user["name"]
 
 
-# Markdown mapping
-def convert_to_zulip_markdown(
-    text: str,
-    users: list[ZerverFieldsT],
-    added_channels: AddedChannelsT,
-    slack_user_id_to_zulip_user_id: SlackToZulipUserIDT,
-) -> tuple[str, list[int], bool]:
-    mentioned_users_id = []
-    text = convert_markdown_syntax(text, SLACK_BOLD_REGEX, "**")
-    text = convert_markdown_syntax(text, SLACK_STRIKETHROUGH_REGEX, "~~")
-    text = convert_markdown_syntax(text, SLACK_ITALIC_REGEX, "*")
-
-    # Map Slack's mention all: '<!everyone>' to '@**all** '
-    # Map Slack's mention all: '<!channel>' to '@**all** '
-    # Map Slack's mention all: '<!here>' to '@**all** '
-    # No regex for this as it can be present anywhere in the sentence
+def convert_slack_workspace_mentions(text: str) -> str:
```

Comment:
I think this commit could still be broken down into parts so that it's much easier to review.

Here's how I'd structure this in three commits:
1. Create and use the `convert_slack_workspace_mentions` helper. Have it be above `convert_to_zulip_markdown` in the file.
2. Move `convert_markdown_syntax` to where you want it in the file - before `convert_to_zulip_markdown` and after the helper you added in the previous commit.
3. Create and use the `convert_slack_formatting` helper. Again, position it in the file where you want it (before `convert_to_zulip_markdown` and after the function you moved in the last commit.

I know it may seem overly simplified, but that would allow me as a reviewer to look at each commit, verify what it does and it's correctness very quickly.

Reply:
Got it, other than the 3 commits you've suggested, I ended up splitting out the changes to reorder existing functions too, those starts with `slack_import: Rearrange  ...`